modules/035-cni-flannel/hooks/set_pod_network_mode.go

```diff
@@ -94,10 +44,6 @@ func setPodNetworkMode(input *go_hook.HookInput) error {
 		}
 	}
 
-	if flannelConfig.PodNetworkMode != "" {
-		podNetworkMode = flannelConfig.PodNetworkMode
-	}
-
 	input.Values.Set("cniFlannel.internal.podNetworkMode", podNetworkMode)
```

Comment:
This could be done in template:
```
  {{- $mode := dict "HostGW" "host-gw" "VXLAN" "vxlan" }}
      mode: {{ get $mode $.Values.cniFlannel.podNetworkMode }}

```

Reply:
it isn't urgent to change.